lua/neo-tree/sources/common/file-items.lua

```diff
@@ -117,7 +117,7 @@ function create_item(context, path, _type, bufnr)
     path = path,
     type = _type,
   }
-  if item.type == "link" then
+  if vim.fn.getftype(path) == "link" then
```

Comment:
I worry that this will have some impact on performance on non-windows computers if we have this extra function call for every file. Can you change it to only do this extra check if the OS is windows?

```suggestion
  if utils.is_windows then
    if vim.fn.getftype(path) == "link" then
      item.type = "link"
    end
  end
  if item.type == "link" then
```

Reply:
Commited. Thank your review.